python/ray/rllib/evaluation/interface.py

```diff
@@ -95,15 +115,7 @@ def set_weights(self, weights):
 
     @DeveloperAPI
     def compute_apply(self, samples):
-        """Fused compute gradients and apply gradients call.
-
-        Returns:
-            info: dictionary of extra metadata from compute_gradients().
-
-        Examples:
-            >>> batch = ev.sample()
-            >>> ev.compute_apply(samples)
-        """
+        """Deprecated: override learn_on_batch instead."""
```

Comment:
any need to log a warning here?

Reply:
It's hard to tell when to warn, since we still fallback to it.